data/json/npcs/TALK_ALLY_TUTORIAL.json

```diff
@@ -73,7 +83,7 @@
   {
     "id": "TALK_ALLY_TUTORIAL_TRAINING",
     "type": "talk_topic",
-    "dynamic_line": "If I'm better at a skill than you, I can help you improve.  But it's boring to teach a lot, so I'm not going to do it very often.  And I'm not going to do it when we're in danger or if I'm hungry or tired or if you're driving.\n  If we're someplace safe and you're reading a book that improves skills, I'll listen if I don't have that skill.  You can even read me books for skills that you already have.",
+    "dynamic_line": "If I'm better at a skill than you, I can help you improve.  But it's boring to teach a lot, so I'm not going to do it very often.  And I'm not going to do it when we're in danger or if I'm hungry or tired or if you're driving.\n  If we're someplace safe and you're reading a book that improves skills, I'll listen if I don't have that skill.  You can even read me books for skills that you already have.\n  If you give me a book that teaches a skill and that's accessible to someone with my current skill, you can ask me about my current activity and tell me to read it, and I'll learn from the book.  But while I'm reading, I won't be following you around, so make sure I'm in a safe place.",
```

Comment:
May want to clarify here, that they can't learn from martial arts books.

Reply:
Fixed.